contrib/node/examples/src/node/hello-test/npm-shrinkwrap.json

```diff
@@ -0,0 +1,1983 @@
+{
```

Comment:
Is there a smaller target you could use as an example? Something with fewer deps?

This seems like overkill.